edx_adapt/tests/logic_tests.py

```diff
@@ -45,7 +45,9 @@ def _answer_pre_assessment_problems(self, correct=0, attention_question=True):
         """
         pre_assessments = ['Pre_assessment_{}'.format(i) for i in range(0, 14)]
         for i, problem in enumerate(pre_assessments):
-            correct_value = True if i < correct - attention_question or (attention_question and i == 13) else False
+            correct_value = (
```

Comment:
But wait, `attention_question` is Boolean and you subtract it from correct? Something is wrong here.